darwin/PlatformHelpers.c

```diff
@@ -95,23 +95,12 @@ double Platform_calculateNanosecondsPerMachTick(void) {
 #else
    mach_timebase_info_data_t info;
 
-   /* WORKAROUND for `mach_timebase_info` giving incorrect values on M1 under Rosetta 2.
-    *    rdar://FB9546856 https://openradar.appspot.com/radar?id=5055988478509056
-    *
-    *    We don't know exactly what feature/attribute of the M1 chip causes this mistake under Rosetta 2.
-    *    Until we have more Apple ARM chips to compare against, the best we can do is special-case
-    *    the "Apple M1" chip specifically when running under Rosetta 2.
-    */
```

Comment:
I have one small advice. Don't remove the original code comment as it would help other programmers understand why the workaround is needed.
（一個小建議，不要隨便移除原程式碼的註解，因為它可以幫助其他程式員快速了解這個workaround出現的原因。）